extensions-contrib/materialized-view-maintenance/src/main/java/io/druid/indexing/materializedview/MaterializedViewSupervisor.java

```diff
@@ -240,11 +239,7 @@ public void reset(DataSourceMetadata dataSourceMetadata)
   }
 
   @Override
-  public void checkpoint(
-      @Nullable String sequenceName,
-      @Nullable DataSourceMetadata previousCheckPoint,
-      @Nullable DataSourceMetadata currentCheckPoint
-  )
+  public void checkpoint(int taskGroupId, DataSourceMetadata previousCheckPoint, DataSourceMetadata currentCheckPoint)
```

Comment:
Need fix code style issues

Reply:
@asdf2014 thanks for the review! Maybe you saw an old commit. I've resolved all code style issues.

Reply:
Um.. I am sure. It still exists in the latest version code. :sweat_smile:

```java
public void checkpoint(int taskGroupId, DataSourceMetadata previousCheckPoint, DataSourceMetadata currentCheckPoint)
```
should be
```java
public void checkpoint(
      int taskGroupId,
      DataSourceMetadata previousCheckPoint,
      DataSourceMetadata currentCheckPoint
)
```

Reply:
I should see the latest version code when i visit [https://github.com/apache/incubator-druid/pull/5996/files](https://github.com/apache/incubator-druid/pull/5996/files) this page, right?

Reply:
Yes, you're seeing the latest one. That line doesn't exceed 120 columns.

Reply:
Yep, you're right.